src/main/kotlin/calculator/Operator.kt

```diff
@@ -0,0 +1,38 @@
+package calculator
+
+import com.sun.org.apache.xpath.internal.operations.Plus
+
+enum class Operator(val op: String) {
+    PLUS("+"),
```

Comment:
Kotlin Lambda식을 활용하면 다음과 같이 작성할 수도 있습니다! 당장은 참고만 해주셔도 좋습니다 🙂 
```suggestion
enum class Operator(val op: String, val operate: (Int, Int) -> Int) {
    PLUS("+", { a, b -> a + b }),
```